MAUI/DataForm/floating-label-layout.md

```diff
@@ -0,0 +1,777 @@
+
+{% tabs %}
+{% highlight XAML %}
+<ContentPage 
+...
+xmlns:dataForm="clr-namespace:Syncfusion.Maui.DataForm;assembly=Syncfusion.Maui.DataForm">
+<dataForm:SfDataForm x:Name="dataForm" 
+                        LayoutType="TextInputLayout"
+                        DataObject="{Binding ContactInfo}"
+                        AutoGenerateItems="False"
+                        GenerateDataFormItem="OnGenerateDataFormItem">
+    <dataForm:SfDataForm.Items>
+        <dataForm:DataFormTextItem FieldName="Name">
+            <dataForm:DataFormTextItem.ValidMessageLabelTextStyle>
+                <dataForm:DataFormTextStyle TextColor="ForestGreen" FontAttributes="Italic"/>
+            </dataForm:DataFormTextItem.ValidMessageLabelTextStyle>
+        </dataForm:DataFormTextItem>
+    </dataForm:SfDataForm.Items>
+</dataForm:SfDataForm>
+</ContentPage>
+{% endhighlight %}
+{% highlight C# %}
+
+this.dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+{
+    if (e.DataFormItem != null && e.DataFormItem.FieldName == "Name")
+    {
+        e.DataFormItem.LayoutType = DataFormLayoutType.TextInputLayout;
+        e.DataFormItem.ValidMessageLabelTextStyle = new DataFormTextStyle { FontAttributes = FontAttributes.Bold, TextColor = Colors.GreenYellow };
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## Appearance customization
+
+#### Changing outline corner radius
+The corner radius of the container will be changed by setting `OutlineCornerRadius` property of `TextInputLayoutSettings` to double value.
+
+{% tabs %}
+{% highlight XAML %}
+<ContentPage 
+...
+xmlns:dataForm="clr-namespace:Syncfusion.Maui.DataForm;assembly=Syncfusion.Maui.DataForm">
+<dataForm:SfDataForm x:Name="dataForm" 
+                        LayoutType="TextInputLayout"
+                        DataObject="{Binding ContactInfo}"
+                        GenerateDataFormItem="OnGenerateDataFormItem">
+    <dataForm:SfDataForm.TextInputLayoutSettings>
+        <dataForm:TextInputLayoutSettings OutlineCornerRadius="10"/>
+    </dataForm:SfDataForm.TextInputLayoutSettings>
+</dataForm:SfDataForm>
+</ContentPage>
+{% endhighlight %}
+{% highlight C# %}
+
+this.dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+{
+    if (e.DataFormItem != null && e.DataFormItem.FieldName == "Name")
+    {
+        e.DataFormItem.LayoutType = DataFormLayoutType.TextInputLayout;
+        e.DataFormItem.TextInputLayoutSettings = new TextInputLayoutSettings
+        {
+            OutlineCornerRadius = 10,
+        };
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Stroke color
+The `Stroke` property of `TextInputLayoutSettings` is used to change the label text and border color of the editors.
```

Comment:
The `Stroke` property of the `TextInputLayoutSettings` changes the label text and border color of the editors.

Reply:
Content changed